HW2/src/main/resources/db/migration/V1__author_table.sql

```diff
@@ -0,0 +1,13 @@
+drop table if exists book;
+drop table if exists author;
+create table author
+(
+    id   bigint generated by default as identity
+        constraint author_pkey
+            primary key,
+    name varchar(255)
+);
+insert into author(name)
```

Comment:
Test data migrations should be placed separately and by default should not be attached to migration process.

Test data should never go in prod.